src/components/button/examples/button-reduce-presence.tsx

```diff
@@ -0,0 +1,97 @@
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Reduce Presence
+ *
+ * This example is identical to the one above, except that here, the
+ * `reducePresence` property has been set to `true`. This will hide the button
+ * when it is disabled. However, it will also make sure that the button remains
+ * visible while the loading animation is ongoing. When the animation is done
+ * and the checkmark has been shown, the button will hide.
+ *
+ * ###### Usage of `reducePresence` property
+ * You normally make a button disabled under two circumstances:
+ * 1. **When the button does not have any functionality at the moment** -
+ * For example a SAVE button which is disabled, when there are no changes to
+ * be saved.
+ * 2. **When the button tries to stop the user from proceeding** –
+ * Like a CONTINUE button in a form or a wizard which stops users from
+ * "continuing" because requirements are not met, required fields are not
+ * filled, or a form has errors.
+ *
+ *:::tip
+ * Sometimes it makes a lot of sense to simply hide a button that *has no
+ * functionality*, instead of leaving it `disabled` but visible.
+ * A good example of such a case is the SAVE and DISCARD buttons in a view.
```

Comment:
> the SAVE and DISCARD buttons in a view

That only makes sense in the context of Lime CRM Web Client. Can we come up with an example that can be understood without that context? Could it just say "the SAVE and DISCARD buttons on a form" instead?

Reply:
By "view" I meant **any** view. But if you think it's less confusing, yes we can use "form" instead. ;)

Reply:
Well, if that's the case, we can just drop the "in a view" part completely:

> A good example of such a case are SAVE and DISCARD buttons.

Reply:
Sure